src/server/database-mock-data.js

```diff
@@ -2,6 +2,7 @@
 
 const initTime = new Date();
 const dayInMs = 1000 * 60 * 60 * 24;
+export const mockUserKey = 'user-8000000000000-uuiduuid-uuid-uuid-uuid-uuiduuiduuid';
 export const mockItems = {
   'item-8523910540005-dd3860f5-b82e-473b-1234-ead0f190b000': {
```

Comment:
what about this form?
[
	{
		key: “”,
		value: {
			name: “”,
			pw: “”
		}
	}
]